darwin/dataset/remote_dataset.py

```diff
@@ -273,9 +269,20 @@ def release(self, name: Optional[str] = None):
         release: Release
             The release created right now
         """
-        self.client.post(f"/datasets/{self.dataset_id}/exports", team=self.team)
+        if annotation_class_ids is None:
+            annotation_class_ids = []
+        payload = {"annotation_class_ids": annotation_class_ids, "name": name}
+        self.client.post(f"/datasets/{self.dataset_id}/exports", payload=payload, team=self.team)
+        time.sleep(2.0) # This is necessary avoid a race condition on pulling the latest release
```

Comment:
Exporting could take > 10 min to run if the dataset is big enough, what we would get back is an in progress export record that isn't very useful. 
Sleeps are in general not a good way to solve this issues

Reply:
Nice catch, I was not entirely sure what was the reason of the race condition. Since we can't know in advance how long is it gonna take and we can't also use a blocking solution (at least yet?) I removed the return statement (which was anyway not used).